include/micm/process/cuda_process_set.cuh

```diff
@@ -4,22 +4,37 @@ namespace micm
 {
   namespace cuda
   {
-    void AddForcingTerms_kernelSetup(
+   double AddForcingTerms_kernelSetup(
         const double* rate_constants_data,
         const double* state_variables_data,
         double* forcing_data,
-        int ngrids,
-        int nrxns,
-        int nspecs,
+        size_t n_grids,
+        size_t n_reactions,
+        size_t n_species,
         const size_t* number_of_reactants,
-        int number_of_reactants_size,
         const size_t* reactant_ids,
-        int reactant_ids_size,
+        size_t reactant_ids_size,
         const size_t* number_of_products,
-        int number_of_products_size,
         const size_t* product_ids,
-        int product_ids_size,
+        size_t product_ids_size,
         const double* yields,
-        int yields_size);
+        size_t yields_size);
+
+    double AddJacobianTerms_kernelSetup(
```

Comment:
```suggestion
    std::chrono::nanoseconds AddJacobianTerms_kernelSetup(
```

Please return a `std::chrono` object, like I suggested above, so that we don't have to keep track of the actual units of time. We can always cast from nanoseconds to microseconds in the driver code and c++ will properly convert for us without us having to really care

Reply:
done.